plugins/modules/content_view_version.py

```diff
@@ -204,6 +204,13 @@ def main():
 
     module.task_timeout = 60 * 60
 
+    if 'version' in module.foreman_params and '.' not in module.foreman_params['version']:
+        try:
+            major_version = int(module.foreman_params['version'])
+            module.foreman_params['version'] = "{0}.0".format(major_version)
```

Comment:
should this branch emit a warning, so the user *knows* their playbook  is not exactly correct?

Reply:
if hammer accepts it we should maybe also.